### PR TITLE
upstream changes

### PR DESCRIPTION
**Pull request by fivetran-jamie**
## PR Overview
**This PR will address the following Issue/Feature:**
https://github.com/fivetran/dbt_apple_store_source/issues/8

**This PR will result in the following new package version:**

v0.3.0  - upstream packages had breaking changes 

**Please detail what change(s) this PR introduces and any additional information that should be known during the review of this PR:**
the country code seed files changed so people need reseed

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [x] dbt compile
- [x] dbt run –full-refresh
- [x] dbt run
- [x] dbt test
- [ ] dbt run –vars (if applicable)

Before marking this PR as "ready for review" the following have been applied:
- [x] The appropriate issue has been linked and tagged
- [x] You are assigned to the corresponding issue and this PR
- [ ] BuildKite integration tests are passing

### Detailed Validation
Please acknowledge that the following validation checks have been performed prior to marking this PR as "ready for review":
- [x] You have validated these changes and assure this PR will address the respective Issue/Feature.
- [x] You are reasonably confident these changes will not impact any other components of this package or any dependent packages.
- [x] You have provided details below around the validation steps performed to gain confidence in these changes.

### Standard Updates
Please acknowledge that your PR contains the following standard updates:
- Package versioning has been appropriately indexed in the following locations:
    - [x] indexed within dbt_project.yml
    - [x] indexed within integration_tests/dbt_project.yml
- [x] CHANGELOG has individual entries for each respective change in this PR
    
- [x] README updates have been applied (if applicable)
    
- [ ] DECISIONLOG updates have been updated (if applicable)
- [x] Appropriate yml documentation has been added (if applicable)

### dbt Docs
Please acknowledge that after the above were all completed the below were applied to your branch:
- [x] docs were regenerated (unless this PR does not include any code or yml updates)

### If you had to summarize this PR in an emoji, which would it be?
 
🧀

### CODE REVIEW

File: packages.yml [hunk 1]
```diff
@@ -1,5 +1,11 @@
...
+# - package: fivetran/google_play
+#   version: [">=0.3.0", "<0.4.0"]
+# - package: fivetran/apple_store
+#   version: [">=0.3.0", "<0.4.0"]
+- git: https://github.com/fivetran/dbt_google_play.git
+  revision: bug/add-alt-country-names
+  warn-unpinned: false
+- git: https://github.com/fivetran/dbt_apple_store.git
+  revision: bug/add-alt-country-names
+  warn-unpinned: false
```
Comment (fivetran-joemarkiewicz) [inline packages.yml:11]: Reminder to update before merge

**Review by fivetran-joemarkiewicz — APPROVED** [review]
This looks good to go pending PR #15 (or the same adjustments) are applied to this PR.

File: README.md [hunk 2]
```diff
@@ -86,7 +86,13 @@ vars:
 
 > 👀 Subscriptions and financial data are NOT included in `app_reporting` data models. This data is leveraged in the individual Google Play and Apple App Store packages, which are installed within the App Reporting package.
 
-## (Recommended) Step 5: Change the Build Schema
+## Step 5: Seed `country_codes` mapping tables (once)
+
+In order to map longform territory names to their ISO country codes, we have adapted the CSV from [lukes/ISO-3166-Countries-with-Regional-Codes](https://github.com/lukes/ISO-3166-Countries-with-Regional-Codes) to align Google and [Apple's](https://developer.apple.com/help/app-store-connect/reference/app-store-localizations/) country name formats for the App Reporting package. 
+
+You will need to `dbt seed` the `google_play__country_codes` [file](https://github.com/fivetran/dbt_google_play_source/blob/main/seeds/google_play__country_codes.csv) and `apple_store_country_codes` [file](https://github.com/fivetran/dbt_google_play_source/blob/main/seeds/apple_store_country_codes.csv) just once.
+
```
Comment (fivetran-reneeli) [inline README.md:94]:
```suggestion
https://github.com/fivetran/dbt_apple_store_source/blob/main/seeds/apple_store_country_codes.csv
```

Comment (fivetran-reneeli) [reply README.md:94]: Think you meant to link the apple codes to this link, not google (404 error)

https://github.com/fivetran/dbt_apple_store_source/blob/main/seeds/apple_store_country_codes.csv

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,9 +1,15 @@
-# dbt_app_reporting v0.UPDATE.UPDATE
+# dbt_app_reporting v0.3.0
 
- ## Under the Hood:
+## Bug Fixes:
+[PR #14](https://github.com/fivetran/dbt_app_reporting/pull/14) includes the following changes:
+- This version of the transform package points to breaking changes in the upstream [Google Play](https://github.com/fivetran/dbt_google_play_source/blob/main/CHANGELOG.md) and [Apple Store](https://github.com/fivetran/dbt_apple_store_source/blob/main/CHANGELOG.md) source packages in which the [country code](https://github.com/fivetran/dbt_apple_store_source/blob/main/seeds/apple_store_country_codes.csv) mapping tables were updated to align with Apple's [format and inclusion list](https://developer.apple.com/help/app-store-connect/reference/app-store-localizations/) of country names.
+  - This is a 🚨**breaking change**🚨 as you will need to re-seed (`dbt seed --full-refresh`) the `google_play__country_codes` [file](https://github.com/fivetran/dbt_google_play_source/blob/main/seeds/google_play__country_codes.csv) and `apple_store_country_codes` [file](https://github.com/fivetran/dbt_google_play_source/blob/main/seeds/apple_store_country_codes.csv) again.
```
Comment (fivetran-reneeli) [inline CHANGELOG.md:6]:
```suggestion
  - This is a 🚨**breaking change**🚨 as you will need to re-seed (`dbt seed --full-refresh`) the `google_play__country_codes` [file](https://github.com/fivetran/dbt_google_play_source/blob/main/seeds/google_play__country_codes.csv) and `apple_store_country_codes` [file](https://github.com/fivetran/dbt_apple_store_source/blob/main/seeds/apple_store_country_codes.csv) again.
```

File: README.md [hunk 4]
```diff
@@ -86,7 +86,13 @@ vars:
 
 > 👀 Subscriptions and financial data are NOT included in `app_reporting` data models. This data is leveraged in the individual Google Play and Apple App Store packages, which are installed within the App Reporting package.
 
-## (Recommended) Step 5: Change the Build Schema
+## Step 5: Seed `country_codes` mapping tables (once)
+
+In order to map longform territory names to their ISO country codes, we have adapted the CSV from [lukes/ISO-3166-Countries-with-Regional-Codes](https://github.com/lukes/ISO-3166-Countries-with-Regional-Codes) to align Google and [Apple's](https://developer.apple.com/help/app-store-connect/reference/app-store-localizations/) country name formats for the App Reporting package. 
+
+You will need to `dbt seed` the `google_play__country_codes` [file](https://github.com/fivetran/dbt_google_play_source/blob/main/seeds/google_play__country_codes.csv) and `apple_store_country_codes` [file](https://github.com/fivetran/dbt_google_play_source/blob/main/seeds/apple_store_country_codes.csv) just once.
```
Comment (fivetran-reneeli) [inline README.md:93]:
```suggestion
You will need to `dbt seed` the `google_play__country_codes` [file](https://github.com/fivetran/dbt_google_play_source/blob/main/seeds/google_play__country_codes.csv) and `apple_store_country_codes` [file](https://github.com/fivetran/dbt_apple_store_source/blob/main/seeds/apple_store_country_codes.csv) just once.
```

File: README.md (hunk 2 above)
Comment (fivetran-jamie) [reply README.md:94]: updated!

**Pull request merged**